eth/api.go

```diff
@@ -678,3 +678,70 @@ func (api *PrivateDebugAPI) GetAccessibleState(from, to rpc.BlockNumber) (uint64
 	}
 	return 0, fmt.Errorf("No state found")
 }
+
+type PublicWemixAPI struct {
+	e *Ethereum
+}
+
+func NewPublicWemixAPI(e *Ethereum) *PublicWemixAPI {
+	return &PublicWemixAPI{e}
+}
+
+func (api *PublicWemixAPI) BriocheConfig() *params.BriocheConfig {
+	return api.e.BlockChain().Config().Brioche
+}
+
+type HalvingInfo struct {
+	HalvingTimes uint64   `json:"halvingTimes"`
+	StartBlock   *big.Int `json:"startBlock"`
+	EndBlock     *big.Int `json:"endBlock"`
+	BlockReward  *big.Int `json:"blockReward"`
```

Comment:
recommend modifying the arguments' types as follows

```suggestion
	HalvingTimes hexutil.Uint64   `json:"halvingTimes"`
	StartBlock   *hexutil.Big `json:"startBlock"`
	EndBlock     *hexutil.Big `json:"endBlock"`
	BlockReward  *hexutil.Big `json:"blockReward"`
```

Ethereum RRC uses  integer types encoded as hex string.
This is to prevent overflows because JSON cast big.Int, uint64, etc. to float64.